### ISSUE TITLE
Accessibility: Cannot move elements with space

### ISSUE THREAD

**Issue by emteknetnz** [dnadesign/silverstripe-elemental#1378]
Moved from https://github.com/silverstripe/silverstripe-elemental/issues/1374

Pressing space on a block to activate keyboard reordering expands the block. This means it is no longer possible to reorder blocks using the keyboard.

~Note: Since this is a bug for existing functionality, it should be fixed in patches where appropriate.~

Update: I don't think this is a regression, changing elemental to 5.0 and running `yarn dev` it still has the same behaviour i.e. space will open the inline edit form for the element, not activate 'move mode'

## PRs
- https://github.com/silverstripe/silverstripe-elemental/pull/1412
- https://github.com/silverstripe/developer-docs/pull/873

**Comment by GuySartorelli** [comment]
PR(s) merged. This will be included in the next minor release.
